A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AttitudeIndicator.js

```diff
@@ -1348,12 +1352,15 @@ class Jet_PFD_AttitudeIndicator extends HTMLElement {
         this.applyAttributes();
     }
     applyAttributes() {
+        let deltaY = this.pitch * this.pitchAngleFactor * this.pitchGradFactor;
```

Comment:
Can be `const`.

Reply:
oh, that's probably what @UncleClapton meant by making things constant - oops.